packages/react-components/react-tree/stories/Tree/TreeExpandIcon.stories.tsx

```diff
@@ -1,7 +1,13 @@
 import * as React from 'react';
-import { Tree, TreeItem, TreeItemLayout, TreeItemValue } from '@fluentui/react-components';
+import {
+  Tree,
+  TreeItem,
+  TreeItemLayout,
+  TreeItemValue,
+  TreeOpenChangeData,
+  TreeOpenChangeEvent,
+} from '@fluentui/react-components';
 import { AddSquare16Regular, SubtractSquare16Regular } from '@fluentui/react-icons';
-import { TreeOpenChangeData, TreeOpenChangeEvent } from './../../src/Tree';
```

Comment:
once we migrate to https://github.com/microsoft/fluentui/pull/31086

importing via relative paths will fail build as it's invalid.